variables/fortuneTeller.js

```diff
@@ -1 +1,6 @@
-// Enter code below
\ No newline at end of file
+// Enter code below
+
+function furtune(numchild,partnersName,geoLocation,jobTitle){
```

Comment:
spaces between arguments would be nicer e.g. `(numchild, partnersName, geoLocation, jobTitle)`